### PR TITLE
OCM-11097 | test: automated id:76093

### PR DESCRIPTION
**Pull request by yuwang-RH**
[OCM-11097](https://issues.redhat.com//browse/OCM-11097) | test: automated id:76093
log: https://privatebin.corp.redhat.com/?0450c771f234d708#CcZhmPiUStFvXPwt5k4Mm7wNqpARXJSp9hkZopFWUjxZ

### CODE REVIEW

File: tests/e2e/test_rosacli_cluster.go [hunk 1]
```diff
@@ -3510,3 +3510,117 @@ var _ = Describe("Sts cluster creation supplemental testing",
...
+				err = clusterService.WaitClusterDeleted(clusterID, 3, 35)
+				Expect(err).To(BeNil())
+			}
+			By("Clean resource")
+			errs := profilehandler.DestroyResourceByProfile(customProfile, rosaClient)
+			Expect(len(errs)).To(Equal(0))
+		})
+
+		It("Create STS cluster with oidc config id but no oidc provider via rosacli in auto mode - [id:76093]",
+			labels.Critical, labels.Runtime.Day1Supplemental,
```
Comment (OAharoni-RedHat) [inline tests/e2e/test_rosacli_cluster.go:3571]: I noticed that the polarion test case for this says it's a Medium importance test, whichever one isn't accurate should be updated

File: tests/e2e/test_rosacli_cluster.go [hunk 2]
```diff
@@ -3510,3 +3510,117 @@ var _ = Describe("Sts cluster creation supplemental testing",
...
+			ocmResourceService rosacli.OCMResourceService
+			testingClusterName string
+		)
+		BeforeEach(func() {
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			clusterService = rosaClient.Cluster
+			ocmResourceService = rosaClient.OCMResource
+
+			By("Prepare custom profile")
```
Comment (OAharoni-RedHat) [inline tests/e2e/test_rosacli_cluster.go:3533]: Is there any rule of thumb on when we choose to create a custom profile like here and when we make a full profile in the yaml file?

File: tests/e2e/test_rosacli_cluster.go (hunk 1 above)
Comment (yuwang-RH) [reply tests/e2e/test_rosacli_cluster.go:3571]: It is a case for cluster creation, I will update the importance in Polarion to Critical.

File: tests/e2e/test_rosacli_cluster.go (hunk 2 above)
Comment (yuwang-RH) [reply tests/e2e/test_rosacli_cluster.go:3533]: A full profile would be created for day-1 cluster creation ,also using for day-2 test case, for each profile should have one CI job using it.
For the test case labeled "labels.Runtime.Day1Supplemental", which will be executed by the supplemental CI job, means no prepared cluster for this kind of cases, it needs to create a cluster on demand, using custom profile is one way to prepare the cluster for it.

Comment (yuwang-RH) [reply tests/e2e/test_rosacli_cluster.go:3533]: By the way, the cases labeled "labels.Runtime.Day1Supplemental" itself should have the full cycle of the created cluster. While the cases with the label like day-2, don't need the cleanup step which will be done by the job step.

Comment (OAharoni-RedHat) [reply tests/e2e/test_rosacli_cluster.go:3533]: Interesting, thank you for the explanation!

**Pull request merged**